changes/ee/fix-13332.en.md

```diff
@@ -0,0 +1,4 @@
+When an S3 Bridge is improperly configured, error messages now contain more informative and easy to read details.
+
+## Breaking changes
```

Comment:
Better move this to `breaking-13332.en.md` so it gets highlighted in the breaking changes section.